toml.md

```diff
@@ -797,7 +797,9 @@ Inline Table
 ------------
 
 Inline tables provide a more compact syntax for expressing tables. They are
-especially useful for grouped data that can otherwise quickly become verbose.
+especially useful for grouped data that can otherwise quickly become verbose or
+(deeply) nested data.
```

Comment:
This is pretty good. Could you make one subtle change to the last line? If you remove the word "data", it would then describe the types of grouped data that inline tables can help to manage.

Reply:
I tweaked the wording now to:

>  They are especially useful for grouped (deeply) nested data that can otherwise quickly
become verbose.

which I felt was a better phrasing and should address your comment too.

Reply:
I must admit that I liked @eksortso 's wording better. "grouped (deeply) nested data" just feels like too many adjectives in a row.

Reply:
I preferred it exactly because it put all adjectives in one place, instead of "amending" it later.

I don't strongly care either way though; I'll leave it as is for now but I'll happily change it if eksortso or other people agree.

Reply:
Arguably, allowing newlines makes inline tables look less crowded, even if they aren't nested. So even with all the adjectives, it gets across the point that this change is about.

Reply:
Not to ignore the "grouped" part. That's also a feature, because the inline syntax puts subtables in the middle of a table block without breaking the block apart. Not being able to do so was an issue that visitors have expressed in the past.